doc/library/config.txt

```diff
@@ -716,6 +716,23 @@ import theano and print the config variable, as in:
 
     Location of the magma library.
 
+.. attribute:: config.ctc.enabled
+
+    Default: ``'False'``
+
+    If ``'True'``, use `warp-ctc <https://github.com/baidu-research/warp-ctc>`_ for CTC
+    loss computations.
+
+    If ``'False'``, disable warp-ctc.
+
+.. attribute:: config.ctc.root
+
+    Default: ``''``
+
+    Location of the warp-ctc folder. The folder should contain a build
+    subfolder with the shared library (libwarpctc.so), and another
```

Comment:
Having a "build" subdirectory is strange. Normally, the subdirectory is named lib or lib64. Can you also support that? Why did you do differently? Normally, when a c packages is installed it is like that.